### PR TITLE
[RHACS] Release 3.69.1

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Include the following updates:
- Release notes 
- Architecture page content and images
- Scanner DB related updates

### CODE REVIEW

File: modules/acs-architecture-overview.adoc [hunk 1]
```diff
@@ -3,102 +3,46 @@
...
-You deploy the per-node components in all nodes that you want to monitor.
-{product-title} includes the following per-cluster components:
+* *Scanner* (only on {ocp}):
+On {ocp}, RHACS installs a lightweight version of Scanner on each secured cluster (_Figure 1_) to enable scanning of images in the integrated OCR.
 
-* Collector
-
-[discrete]
-=== Collector
+Figure 2 depicts the architecture in a non-OCP Kubernetes environment, where you only install the Scanner centrally.
```
Comment (kalexand-rh) [inline modules/acs-architecture-overview.adoc:45]:
```suggestion
Figure 2 depicts the architecture in a non-OCP Kubernetes environment, where you install only the Scanner centrally.
```

Please double-check your use of "only" throughout. It's a subtle thing, but "only" is supposed to go close to the thing you're limiting.

File: modules/acs-quick-install-secured-cluster-using-helm.adoc [hunk 2]
```diff
@@ -27,5 +27,17 @@ $ helm install -n stackrox --create-namespace \
     --set clusterName=<name_of_the_secured_cluster> \
     --set centralEndpoint=<endpoint_of_central_service> <2>
 ----
+
+* Run the following command on OpenShift clusters:
```
Comment (kalexand-rh) [inline modules/acs-quick-install-secured-cluster-using-helm.adoc:31]:
```suggestion
* Run the following command on {ocp} clusters:
```

**Review by kalexand-rh — COMMENTED** [review]
I have a few picks for now and suggestions for later, but this is looking good.

File: release_notes/369-release-notes.adoc [hunk 3]
```diff
@@ -77,9 +91,11 @@ Red Hat is deprecating some of the features in {product-title} 3.69. Red Hat wil
 
 * {product-title} 3.70.0:
 
-** {product-title} will not allow deleting default policies. So instead of deleting, you can disable default policies that you do not need.
+** {product-title} will not allow deleting default policies. So rhather than deleting, you can disable default policies that you do not need.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/369-release-notes.adoc:94]: This is for the next version, hence will.

File: release_notes/369-release-notes.adoc [hunk 4]
```diff
@@ -7,23 +7,37 @@ toc::[]
 
 {product-title} ({product-title-short}) 3.69 includes feature enhancements, bug fixes, scale improvements, and other changes.
 
-*Release date:* March 21, 2022
-
 [id="new-features"]
 == New features
 
+[id="released-in-version-3691"]
+=== Released in version 3.69.1
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/369-release-notes.adoc:14]: Thank you, yes this is correct.

**Pull request merged**